public/scripts/worktrollyUI.js

```diff
@@ -8,26 +8,30 @@ $("#createNewTaskButton").click(function() {
 
 $("#backToMainFromCreateNewTask").click(function() {
     newPage = "bodyContent";
-    uiLoader();    
+    uiLoader();
 });
 
 $("#bodyContent").ready(function() {
     var result = getTasks();
     console.log(result);
 });
 
-$("#createNewTask").ready(function(){
+$("#createNewTask").ready(function() {
     $("#createNewTask").hide(0);
 });
 
-$("#editTask").ready(function(){
+$("#editTask").ready(function() {
     $("#editTask").hide(0);
 });
 
-$("#logWork").ready(function(){
+$("#logWork").ready(function() {
     $("#logWork").hide(0);
 });
 
+function getTaskId() {
+    return ++initialTaskNumber;
+}
+
```

Comment:
This method must have implementation inside firebase functions. Otherwise it will not work, Also the implementation must changes.

At first based on the category the the total task will be fetched from db.

example:

if new task is from the development category, then.

totalDevelopmentTask must be fetched from the Main/RawData in cloud firestore.

then the number must be incremented by one and stored back in same location.
and the new number will be added: "D"+newNumberformed

and this is the taskId.

Reply:
I have tried to implement this
I'm still trying to complete this.